Client/mods/deathmatch/logic/luadefs/CLuaVehicleDefs.cpp

```diff
@@ -4273,3 +4275,30 @@ std::variant<bool, std::array<CVector, 4>> CLuaVehicleDefs::OOP_GetVehicleEntryP
 
     return entryPoints;
 }
+
+bool CLuaVehicleDefs::AddVehicleSirens(CClientVehicle* vehicle, std::uint8_t sirenType,  std::uint8_t sirenCount, std::optional<bool> enable360, std::optional<bool> enableLOSCheck, std::optional<bool> enableRandomiser, std::optional<bool> enableSilent) noexcept
+{
+    eClientVehicleType vehicleType = vehicle->GetVehicleType();
+
+    if (vehicleType != CLIENTVEHICLE_CAR && vehicleType != CLIENTVEHICLE_MONSTERTRUCK && vehicleType != CLIENTVEHICLE_QUADBIKE &&  vehicleType != CLIENTVEHICLE_TRAIN)
```

Comment:
Please, remove an extra space between `&&` and `vehicleType != CLIENTVEHICLE_TRAIN`.